ui/flow.reel/flow.meta

```diff
@@ -1,7 +1,7 @@
 {
     "blueprint_flow_stride": {
         "prototype": "core/meta/property-blueprint",
```

Comment:
Maybe worth updating property-blueprint to property-descriptor at the same time?